app/assets/stylesheets/govuk_publishing_components/components/_search.scss

```
@@ -18,7 +19,7 @@ $large-input-size: 50px;
   .js-enabled & {
     position: absolute;
     left: govuk-spacing(3);
```

We've lost the spacing to the left of the label text when it's linearised - would this help fix it?

```suggestion
    left: 0;
    padding-left: govuk-spacing(3);
```

---

Good spot, thanks.